### PR TITLE
feat(Add Source): WIP

### PR DESCRIPTION
**Pull request by K1ngfish3r**
Currently in PR
NobleMTL
PandaMTL
LightNovelPub
LNMTL
NovelUpdates WIP (need hardcode fix for chapter listing due to redirects)
Wuxiaworld Broken (requires grpc fetch)

### CODE REVIEW

File: libs/showToast.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+async function showToast(message) {
```
Comment (nyagami) [inline libs/showToast.js:1]: oh, showToast is synchronous function. So no need `async` keyword

File: plugins/english/foxaholic.js [hunk 2]
```diff
@@ -44,76 +44,79 @@ async function parseNovelAndChapters (novelUrl){
...
-      .find('.summary-content')
-      .text()
-      .replace(/[\t\n]/g, '')
       .trim();
+    const detail = loadedCheerio(this).find('.summary-content').html();
 
     switch (detailName) {
       case 'Genre':
-        novel.genre = detail.trim().replace(/[\t\n]/g, ',');
+        novel.genre = loadedCheerio(detail)
```
Comment (nyagami) [inline plugins/english/foxaholic.js:60]: `novel.genres`

same for the others

File: libs/showToast.js (hunk 1 above)
Comment (K1ngfish3r) [reply libs/showToast.js:1]: i remove async

File: plugins/english/foxaholic.js (hunk 2 above)
Comment (K1ngfish3r) [reply plugins/english/foxaholic.js:60]: i change all mention of ``genre`` to ``genres``

**Pull request merged**